src/main/java/Duke.java

```diff
@@ -1,10 +1,160 @@
+import java.util.ArrayList;
+import java.util.Scanner;
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
+    public static String printIntro() {
+        return " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+                + "|____/ \\__,_|_|\\_\\___|\n" + "____________________________________________________________\n" +
+                " Hello! I'm Duke\n" +
+                " What can I do for you?\n" +
+                "____________________________________________________________\n";
+    }
+
+    public static String printOutro() {
+        return "____________________________________________________________\n" +
+                " Bye. Hope to see you again soon!\n" +
+                "____________________________________________________________\n";
+    }
+
+    public static String list(ArrayList<Task> a, int counter) {
+        String x = ("Here are the tasks in your list: " + "\n" +
+                "____________________________________________________________\n");
+        String result = "";
+        for (int i = 0; i < counter; i++) {
+            result += (i + 1) + ". " + a.get(i).toString() + "\n";
+        }
+        String y = ("____________________________________________________________\n");
+        return x + result + y;
+    }
+
+    public static String printMark(Task task) {
+        if (task.getStatus()) {
+            return "____________________________________________________________\n" +
+                    "Nice! I've marked this task as done:" + "\n" + task.toString() + "\n" +
+                    "____________________________________________________________\n";
+        } else {
+            return "____________________________________________________________\n" +
+                    "OK, I've marked this task as not done yet-:" + "\n" + task.toString() + "\n" +
+                    "____________________________________________________________\n";
+        }
+    }
+
+    public static String added(Task task, int count) {
```

Comment:
```suggestion
    public static String printAdded(Task task, int count) {
```

Methods should have verbs as names.